audits/mobile-friendly/viewport.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+class Viewport {
+
+  static get tags() {
+    return ['Mobile Friendly'];
+  }
+
+  static get description() {
+    return 'Site has a viewport meta tag';
+  }
+
+  static audit(inputs) {
+    const viewportExpression = /<meta ([^c]+)content="width=/gim;
```

Comment:
this seems like it would rarely fail, but is strictly less powerful than the old version. E.g. string in a <pre> could give it trouble